mycroft/client/speech/hotword_factory.py

```diff
@@ -60,6 +65,13 @@ def msec_to_sec(msecs):
 
 
 class HotWordEngine:
+    """Hotword/Wakeword base class to be implemented by all wakeword plugins.
+
+    Arguments:
+        key_phrase (str): string representation of the wakeword
+        config (dict): Configuration block for the specific wakeword
+        lang (str): language code (BCP-47)
+    """
```

Comment:
It seems we more often use the two word version: "wake word" so be good to be consistent.